README.md

```diff
@@ -72,6 +75,10 @@ yarn electron package:preview
 
 The packaged application is located in `applications/electron/dist`.
 
+**Known Issue** 
+It is possible that during the `package:preview` step users encounter an error, see [#132](https://github.com/eclipse-theia/theia-blueprint/issues/132). 
```

Comment:
Now that we understand the core issue (https://github.com/eclipse-theia/theia-blueprint/issues/138), it's much easier for someone to find-out about it from the `Issues` tab.  So,  I do not think it's worth mentioning in the README, anymore. WDYT?

Reply:
I have a PR that should fix the issue, if you want to give it a try locally: https://github.com/eclipse-theia/theia-blueprint/pull/139

Reply:
i ll report my findings in #139. As for the location, as a new contributor I would expect no errors upon following the documented building procedure. Known issues with compiling the code should be mentioned alongside the documentation for compiling i.e. the `readme`, I believe it it is not a realistic expectation for folks to read issues first and then proceed with cloning / forking.  Feel free to remove the `known issue` if it is no longer relevant. But for now i d rather keep that on the readme.

Reply:
> Known issues with compiling the code should be mentioned alongside the documentation for compiling i.e. the readme

Makes sense, when explained like this. Indeed a build issue is "special". 

With #139 now merged, do you think we still need this entry? If not please remove it, and in that case I would also remove the section, until we have something to document in it.